modules/nf-core/arcashla/extract/meta.yml

```diff
@@ -0,0 +1,59 @@
+name: "arcashla_extract"
+description: Extracts reads mapped to chromosome 6 and any HLA decoys or chromosome 6 alternates.
+keywords:
+  - HLA
+  - genotype
+  - RNA-seq
+tools:
+  - "arcashla":
+      description: "arcasHLA performs high resolution genotyping for HLA class I and class II genes from RNA sequencing, supporting both paired and single-end samples."
+      homepage: "https://github.com/RabadanLab/arcasHLA"
+      documentation: "https://github.com/RabadanLab/arcasHLA"
+      tool_dev_url: "https://github.com/RabadanLab/arcasHLA"
+      doi: "10.1093/bioinformatics/btz474"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM file
+      pattern: "*.bam"
```

Comment:
Any preconditions for the genome reference for this BAM file?

Note also from the docs:
> If the BAM file is not indexed, this tool will run samtools index before extracting reads.

Reply:
Hi @heuermh, I found a comment on that: https://github.com/RabadanLab/arcasHLA/issues/62#issuecomment-799757778. Do you think we should add anything on that here?

I will add the remark you quoted, thanks.

Reply:
Thanks for the link!  Perhaps a succinct rewording might work well here.

Reply:
I added the statement on the indexing. Let me know when you think we should add more here.